### PR TITLE
Use String#repeat

### PR DESCRIPTION
**Pull request by shvaikalesh**

### CODE REVIEW

File: toRoman.js [hunk 1]
```diff
@@ -1,67 +1,54 @@
...
-    }
+    return romanify10(num)
 
     function romanify10(num) {
         if (num < 10) {
             if (num == 0) return ''
-            else if (num < 4)   num = multiply(num, 'I')
-            else if (num == 4)  num = 'IV'    
-            else if (num < 9)   num = 'V' + multiply((num - 5), 'I' ) 
-            else if (num == 9)  num = 'IX'
```
Comment (shvaikalesh) [inline toRoman.js:21]: to get rid of trailing spaces, try http://editorconfig.org/

File: toRoman.js [hunk 2]
```diff
@@ -1,67 +1,54 @@
...
+            else if (num < 4)   num = 'I'.repeat(num)
+            else if (num == 4)  num = 'IV'
+            else if (num < 9)   num = 'V' + 'I'.repeat(num - 5) 
+            else if (num == 9)  num = 'IX'
         }    
         return num
     }
 
     function romanify100(num) {
-        num += ''
```
Comment (shvaikalesh) [inline toRoman.js:27]: `+= ''` was really awesome in ES5 times. However it does not handle objects (edge case, ha?) correctly.

``` js
let obj = { [Symbol.toPrimitive]: hint => hint }
obj + '' // => 'default', not quite what you need
`${obj}` // => 'string', but Babel compiles wrong
String(obj) // => 'string', but does not throw for symbols
''.concat(obj) // => 'string', but 'default' in older V8s (node 0.10)
```

File: toRoman.js [hunk 3]
```diff
@@ -1,67 +1,54 @@
...
           , rest = num.join('')
 
-        first = multiply(first, 'M')
+        first = 'M'.repeat(first)
 
         return first + romanify1000(rest)
     }
-}
\ No newline at end of file
+}
```
Comment (shvaikalesh) [inline toRoman.js:54]: Another thing `.editorconfig` will help with

**Comment by 0xfirelight** [comment]
thnx!

**Pull request merged**